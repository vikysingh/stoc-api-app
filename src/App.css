@import url('https://fonts.googleapis.com/css2?family=Mulish&display=swap');

body {
  margin: 0;
  padding: 0;
  max-width: 100vw;
  overflow-x: hidden;
  font-family: 'Mulish', sans-serif;
  background-color: #181926;
}

* {
  text-decoration: none;
}