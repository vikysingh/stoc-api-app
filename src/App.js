import React from 'react'
import { Switch, BrowserRouter, Route } from "react-router-dom"

import './App.css'
import Dashboard from "./components/Dashboard/Dashboard"
import Home from "./components/Home/Home"
import { ViewContextProvider } from "./context"


function App() {
  return (
    <BrowserRouter>
      <Switch>
        <div className="App">
          
          <Route exact path="/">
            <Home />
          </Route>

          <Route path="/stocks">
            <ViewContextProvider>
              <Dashboard />
            </ViewContextProvider>
          </Route>

        </div>
      </Switch>
    </BrowserRouter>
  );
}

export default App;
