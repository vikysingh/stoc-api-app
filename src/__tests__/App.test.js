import React from "react"
import App from "../App"
import { create } from "react-test-renderer"

describe("My first snapshot text", () => {
    test("testing app", () => {
        let tree = create(<App />)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})