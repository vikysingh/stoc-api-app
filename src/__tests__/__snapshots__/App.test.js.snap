// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first snapshot text testing app 1`] = `
<div
  className="App"
>
  <header
    className="header"
    role="banner"
  >
    <h1>
      Top 15 stocks
    </h1>
    <button
      className="primary_btn"
    >
      <a
        href="/stocks"
        onClick={[Function]}
      >
        Access stocks!
      </a>
    </button>
  </header>
</div>
`;
